custom_components/contact_energy/sensor.py

```diff
@@ -305,7 +282,6 @@ def make_attribute(self, response, value):
             else:
                 newPoint['price'] = 0
             return newPoint
-
```

Comment:
There shouldn't be any comment marks (''') at the end of the file as there is no starting comment marks.

Reply:
See my above comment, I messed up adding back in the commented out code.